pkgs/applications/audio/zam-plugins/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchgit, boost, libX11, mesa, liblo, jack2, ladspaH, lv2, pkgconfig, rubberband, libsndfile }:
+
+stdenv.mkDerivation rec {
+  name = "zam-plugins";
+  rev = "f75581f76d3e0acd8b604c07ff1e69a649b505ff";
```

Comment:
Why is this using an arbitrary commit instead of a stable release(https://github.com/zamaudio/zam-plugins/releases)?, also there is no need to declare `rev` outside of `src`.

You can also set the `PREFIX` in `makeFlags` instead of using sed to patch the makefile